components/language-switcher/readme.md

```diff
@@ -0,0 +1,8 @@
+A component for changing the language of the current page.
+
+### Notes
+- Assumes the first path in the page URL immediately after hostname is a valid locale code e.g. https://www.mozilla.org/en-US/firefox/.
+- This molecule can be displayed inside the Footer organism, but could also be used elsewhere if more appropriate.
```

Comment:
Let's rewrite without the atomic design lingo.
```suggestion
- This component can be displayed inside the Footer component, but could also be used elsewhere if more appropriate.
```

Reply:
d'oh! good catch.